tests/snuba/test_snuba.py

```diff
@@ -97,37 +97,6 @@ def test_organization_retention_larger_than_end_date(self):
                 },
             ) == {}
 
-    def test_use_group_id(self):
-        base_time = datetime.utcnow()
-        group = self.create_group()
-        self._insert_event_for_time(base_time, group_id=group.id)
-
-        with self.options({'snuba.use_group_id_column': True}):
-            # verify filter_keys and aggregation
-            assert snuba.query(
-                start=base_time - timedelta(days=1),
-                end=base_time + timedelta(days=1),
-                groupby=['issue'],
-                filter_keys={
-                    'project_id': [self.project.id],
-                    'issue': [group.id]
```

Comment:
We should remove `issue` as an alias for `group_id` in Snuba.

Reply:
+100